sirepo/util.py

```diff
@@ -130,6 +131,18 @@ def __init__(self, display_text, *args, **kwargs):
             **kwargs
         )
 
+# python 2/3 string and bytes compatibility
+if sys.version_info < (3,):
```

Comment:
feature test str.decode. Bad example for others to follow. Move this to pkcompat.